src/components/AddressSearch/index.tsx

```diff
@@ -411,10 +434,18 @@ function AddressSearch(
                         }}
                         styles={{
                             textInputContainer: [styles.flexColumn],
-                            listView: [StyleUtils.getGoogleListViewStyle(displayListViewBorder), styles.overflowAuto, styles.borderLeft, styles.borderRight, !isFocused && {height: 0}],
+                            listView: [
+                                StyleUtils.getGoogleListViewStyle(displayListViewBorder),
+                                getListViewHeight(),
```

Comment:
There's no need for this to be a function I think
`getListViewOverflow` file can be renamed to `listViewOverflow` and instead of a function, 
`index.js` file will just return `styles.overflowAuto` and
`index.native.js` will return `styles.overflowHidden`